documentation/sdk/docs/how_tos/outputs/output_unlock_conditions.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: Output Unlock Conditions
+description: 'How to create ouputs with different unlock conditions'
```

Comment:
Why do I think about "How to create Aua" when reading this? :trollface:
Joke, but I think it generally doesn't sound that great, how about this? Or does it even need to have "How to"?
```suggestion
description: 'How to set unlock conditions for outputs'
```